cloud/amazon/ec2_asg.py

```diff
@@ -112,8 +112,8 @@
     version_added: "2.0"
   wait_timeout:
     description:
-      - how long before wait instances to become viable when replaced.  Used in concjunction with instance_ids option.
-    default: 300
+      - The number of seconds to wait for instances to become healthy after they are replaced.  Used in conjunction with the replace_instances option.
+    default: 300 seconds
```

Comment:
@axilleas I was just copying the style that appears in the rest of the documentation for this module, e.g. for 'default_cooldown' and 'health_check_period'